### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by DavidHaslam** [ThomHehl/Moffatt#4]
### Suggestions:
1. Add link for further information about the [Moffat Bible](https://en.wikipedia.org/wiki/Moffatt,_New_Translation) translation.
2. Provide a clear statement about the **copyright** status.
The Complete Bible was first published in 1922, so even in the USA under the [DCMA](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act) and the [CTEA](https://en.wikipedia.org/wiki/Copyright_Term_Extension_Act), it should be public domain.
3. Clarify which **edition** was used as the primary text source and proofchecking reference.

**Comment by DavidHaslam** [comment]
Some further information about the Moffatt 1922 edition is hosted [here](http://innvista.com/culture/religion/bible/versions/moffatt-new-translation/).

There's evidently a copy of the 1922 edition at [Tyndale House, Cambridge](http://www.tyndale.cam.ac.uk/). 

**Tyndale House** is the home of [STEP Bible](http://www.stepbible.com/) a project based on the JSword API. _See also [Step2011](http://www.tyndale.cam.ac.uk/step2011)_.

btw. The director of **STEP Bible** (Dr David Instone-Brewer) is one of my friends.

**Comment by DavidHaslam** [comment]
### Why we should be critical of Moffatt

> Moffatt's departed from traditional translations in several areas. First, he held to the [documentary hypothesis](https://en.wikipedia.org/wiki/Documentary_hypothesis) and printed his Bible in different typefaces according to which author he believed had written each particular section.

The DH should have been "put out to grass" decades ago. It's presuppositions have been entirely discredited. For a major work that demolishes the whole structure, see the three volume _magnum opus_ by **Kenneth Kitchen** and **Paul Lawrence**, [Treaty, Law and Covenant in the Ancient Near East](https://www.amazon.co.uk/Treaty-Covenant-Ancient-Near-East/dp/3447067268).

**Comment by ThomHehl** [comment]
Yes, the DH is crap and the Moffatt was widely criticized for it, but it only effects the order of chapters and verses and stuff. His translation is still one of the best for his Old Testament.

**Comment by DavidHaslam** [comment]
I've just updated **README.me** in my branch by adding two links and some information about the copyright status.
